infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -217,7 +217,7 @@ public LogReplicationEntry resend() {
             log.warn("Caught a timeout exception ", e);
             force = true;
         } catch (Exception e) {
-            log.warn("Caught an Exception and will notify discovery service ", e);
+            log.trace("Caught an Exception and will notify discovery service ", e);
```

Comment:
I'm curious why is this noisy? I mean why do you see exceptions so often? is it just because you're exercising lock changes and it should fail or some underlying bug we should look into?

Reply:
It's noisy on the lock changes when the timeouts start happening. Did not look like a bug to me.